dev-db/firebird/firebird-2.5.3.26780.0-r3.ebuild

```diff
@@ -20,7 +20,7 @@ SLOT="0"
 KEYWORDS="~amd64 ~x86"
 
 IUSE="debug doc client examples superserver xinetd"
-REQUIRED_USE="^^ ( client superserver xinetd )"
+REQUIRED_USE="?? ( client superserver xinetd )"
```

Comment:
Not quite, since "^^" means "exactly one of" and a SuperClassic server build requires all these USE flags disabled ("-client -superserver -xinetd").

With "^^" this results in the following error from portage:
The following REQUIRED_USE flag constraints are unsatisfied:
    exactly-one-of ( client superserver xinetd )

With "??" it builds fine.